src/components/AddBook.js

```diff
@@ -2,6 +2,7 @@
 import { useSelector, useDispatch } from 'react-redux';
 import { fetchGetBooks, fetchPostBook } from '../redux/books/booksSlice';
 import { v4 as uuidv4 } from 'uuid';
```

Comment:
- Hi, I would advise you not to disable linters, you can solve the bugs manually. 
- ESLint is a tool for identifying and reporting on patterns found in ECMAScript/JavaScript code, with the goal of making code more consistent and avoiding bugs.
![Capture](https://user-images.githubusercontent.com/63972670/224367433-8a854f81-6248-443b-a2e5-364b59181236.PNG)